packages/tools/client-debugger/client-debugger-view/src/components/RenderLabel.tsx

```diff
@@ -0,0 +1,72 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import React from "react";
+import { Primitive } from "@fluid-tools/client-debugger";
+// eslint-disable-next-line import/no-internal-modules
+import { TreeItemLayout } from "@fluentui/react-components/unstable";
+import { tokens } from "@fluentui/react-components";
+
+/**
+ * Input props to {@link RenderLabel}
+ */
+export interface RenderLabelProps {
+	/**
+	 * Key of the child node from Record {@link VisauTree}.
+	 */
+	label: string;
+
+	/**
+	 * Type of the object.
+	 */
+	nodeTypeMetadata?: string | undefined;
+
+	/**
+	 * Nodekinds to filter rendering pattern in {@link TreeDataView}.
+	 */
+	nodeKind?: string;
+
+	/**
+	 * Size of the children inside the data.
+	 */
+	itemSize?: Primitive;
+
+	/**
+	 * Primitive value of the node if node is {@link VisualNodeKind.FluidValueNode} or {@link VisualNodeKind.ValueNode}
+	 */
+	nodeValue?: Primitive;
+}
+
+/**
+ * Renders the header of the item.
+ */
+export function RenderLabel(props: RenderLabelProps): React.ReactElement {
```

Comment:
Nit: Since this component is intended to be used as the header in our tree view (we even use TreeItemLayout, etc. below), what do you think about calling this something like "TreeHeader" or something?

Reply:
`TreeHeader` sounds much better. Thanks!